### PR TITLE
fixed footer

### PR DESCRIPTION
**Pull request by yohanym95**
added correct links to footer.

### CODE REVIEW

**Comment by yohanym95** [comment]
@gamage1995  and @jayasanka-sack please review this.

**Comment by jayasanka-sack** [comment]
looks okay for me

**Comment by Piumal1999** [comment]
Yeah. Its alright

**Review by gamage1995 — COMMENTED** [review]
thanks for the update @yohanym95 :)

@jayasanka-sack are we switching the blog link from medium ?

**Comment by jayasanka-sack** [comment]
Let's go with medium for now. 
But need to fix blog page ASAP. 

@Piumal1999 approve the review request & merge this.

File: assets/content/footer.html [hunk 1]
```diff
@@ -111,23 +111,23 @@ <h5 class="heading">About</h5>
...
-                            <a href="https://sefglobal.org/about.html" class="nav-link">About</a>
+                            <a href="https://sefglobal.org/about/about.html" class="nav-link">About</a>
                         </li>
                         <li class="nav-item">
                             <a href="https://www.youtube.com/channel/UClw7QbeW_FOJz_fDMvjXsJw/" class="nav-link"
                                target="_blank">Portfolio</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://medium.com/@SustainableEduF" class="nav-link" target="_blank">Blog</a>
+                            <a href="https://sefglobal.org/blog.html" class="nav-link">Blog</a>
```
Comment (Piumal1999) [inline assets/content/footer.html:121]: Lets use medium link just for now.

**Review by Piumal1999 — APPROVED** [review]

**Pull request merged**